architecture/users/readme.md

```diff
@@ -56,12 +50,12 @@
             "name" : "..."
         },
         ...
-    ]    
+    ]
     ```
 
-### USR-F2 Я как пользователь хочу иметь возможность получить пользователей по их uuid
+### USR-F2 I, as a user, want to be able to get list of students on their uuid
 
-* USR-F2-D1 На входе - массив uuid, на выходе - массив users с полями uuid, slack, skype и name.
+* USR-F2-D1 At the input there is an uuid array, at the output there is an array of students with the uuid, slack, skype and name fields.
```

Comment:
~At the input there is~ **Input - ** an uuid array, ~at the~ output **-** ~there is~ an array of students with the uuid, slack, skype and name fields.